python/huggingfaceserver/tests/test_model.py

```diff
@@ -32,6 +33,14 @@
 import torch.nn.functional as F
 
 
+# Since vllm must support Python 3.8, we can't use str.removeprefix(prefix)
+# introduced in Python 3.9
```

Comment:
We don't support python 3.8 anymore. I think it is safe to use.

Reply:
@sivanantha321 thanks for confirming that!